canonical-web-and-design/netplan.io
# Fix definitions lists

## Done

The markdown renderer used did not render definition lists, so an additional plugin was installed to deal with these.
Additionally, swapped to another ToC plugin for improved features and security.

## QA
- Check out this branch
- `dotrun clean && dotrun`
- Go to the references page and check that definition lists are rendered correctly. 
E.g. compare https://netplan.io/reference/#authentication with http://localhost:8024/reference/#authentication (you can see the render problem in https://staging.netplan.io/reference/#authentication)
